ext/reflection/php_reflection.c

```diff
@@ -1321,10 +1321,21 @@ PHPAPI void zend_reflection_class_factory(zend_class_entry *ce, zval *object)
 }
 /* }}} */
 
+/* {{{ reflection_extension_factory_ex */
+static void reflection_extension_factory_ex(zval *object, struct _zend_module_entry *module)
+{
+	reflection_instantiate(reflection_extension_ptr, object);
+	reflection_object *intern = Z_REFLECTION_P(object);
+	intern->ptr = module;
+	intern->ref_type = REF_TYPE_OTHER;
+	intern->ce = NULL;
+	ZVAL_STRINGL(reflection_prop_name(object), module->name, strlen(module->name));
```

Comment:
I'm aware this is pre-existing code, but:
```suggestion
	ZVAL_STRING(reflection_prop_name(object), module->name);
```
is a tad bit cleaner

Reply:
done